From be0b142768af364504271e07723176130d4159e2 Mon Sep 17 00:00:00 2001
From: Jian <Jian.Li@freescale.com>
Date: Fri, 20 Mar 2015 15:05:01 +0800
Subject: [PATCH] gstplaybin remove native video  and deinterlace flag

- remove default natvie video anddeinterlace flag
  in playbin for i.MX SoCs

Upstream Status: Inappropriate [platform specific]

Signed-off-by: Jian <Jian.Li@freescale.com>
---
 gst/playback/gstplaybin2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/playback/gstplaybin2.c b/gst/playback/gstplaybin2.c
index 2793eea..8107f3d 100644
--- a/gst/playback/gstplaybin2.c
+++ b/gst/playback/gstplaybin2.c
@@ -510,7 +510,7 @@ struct _GstPlayBinClass
 #define DEFAULT_SUBURI            NULL
 #define DEFAULT_SOURCE            NULL
 #define DEFAULT_FLAGS             GST_PLAY_FLAG_AUDIO | GST_PLAY_FLAG_VIDEO | GST_PLAY_FLAG_TEXT | \
-                                  GST_PLAY_FLAG_SOFT_VOLUME | GST_PLAY_FLAG_NATIVE_VIDEO
+                                  GST_PLAY_FLAG_SOFT_VOLUME
 #define DEFAULT_N_VIDEO           0
 #define DEFAULT_CURRENT_VIDEO     -1
 #define DEFAULT_N_AUDIO           0
-- 
1.7.9.5

